### PR TITLE
website fixes 

### PR DESCRIPTION
**Pull request by danielogen**
#### Summary of Changes (What does this PR do?)
- This PR fixes database connection error generated when the user attempts to access the website using www.airqo.net or www.airqo.africa. 
- Fix `NAN` and `Last Updated 50 year ago` error on website 
- Fix access to `https://airqo.org`
- Fix `wehubit` logo

#### Is this change ready to hit production in its current state?
- [X] Yes
- [X] No

#### Status of maturity (all need to be checked before merging):

- [X] I consider this code done
- [X] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Just head over to production and try accessing the website using www.airqo.net or www.airqo.africa. You should not see any database connection error .

#### What are the relevant tickets?
- [ticket_id]()

#### Screenshots (optional)

### CODE REVIEW

File: web/environments.php [hunk 1]
```diff
@@ -6,10 +6,18 @@
         define('ENVIRONMENT', 'production');
         break;
```
Comment (Baalmart) [inline web/environments.php:7]: Thanks @danielogen , could we be having a default configuration for cases that we might not be able to capture here? Can those cases be handled from here or we have to explicitly look out for them? Thanks.

**Review by Baalmart — COMMENTED** [review]
All looks good, good stuff ....neat too!  Just left some few comments for your review... @danielogen

File: web/environments.php (hunk 1 above)
Comment (danielogen) [reply web/environments.php:7]: Thank you @Baalmart , the default case is actually the `development` environment

**Comment by danielogen** [comment]
@Baalmart, I have now added the fix for `NaN` and `Last Refresh 90 years....` in this PR. You might have to re-review. Thank you for the early feedback.

**Review by NoahNsimbe — APPROVED** [review]
LGTM!

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**